NGUInjector/Managers/ZoneHelpers.cs

```diff
@@ -97,18 +97,21 @@ internal static void OptimizeITOPOD()
             var controller = Main.Character.adventureController;
             var level = controller.itopodLevel;
             var optimal = CalculateBestItopodLevel();
-            if (level == optimal) return;
-            controller.itopodStartInput.text = optimal.ToString();
-            if (optimal == Main.Character.adventure.highestItopodLevel)
+            if (optimal == Main.Character.adventure.highestItopodLevel && optimal != Main.Character.adventureController.maxItopodLevel())
             {
-                optimal++;
+                controller.itopodStartInput.text = optimal.ToString();
+                controller.itopodEndInput.text = Main.Character.adventureController.maxItopodLevel().ToString();
                 Main.Log("Pushed optimal floor up by 1 because highest floor is hit");
+                controller.verifyItopodInputs();
```

Comment:
Without reset (`controller.zoneSelector.changeZone(1000);`) on that path you may end starting 100 levels lower..